Eatery/Controllers/Eateries/Collegetown/CollegetownEateriesViewController.swift

```diff
@@ -48,6 +48,17 @@ class CollegetownEateriesViewController: EateriesViewController {
         }
     }
 
+    override func pushMapViewController() {
```

Comment:
Same code for CollegetownEateries and CampusEateries, is it possible to make eateries part of EateriesViewController and provide an implementation for this function there?